Hello Microverse Project Setup

Rysth opened the pull request:
### Features Implemented 
**Date:** 30/05/2023

- Edited the README.md document with more descriptive text.
- Added an HTML file called index.html
- Added a CSS file called style.css inside of css/style.css.
- Ignored a file called test.md

levy002 reviewed (changes requested):
## Hello @Rysth    , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Highlinters
- Correct git-flow :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Well-detailed PR summary :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make sure to update your PR title by proving a well-detailed title that makes your PR clean and professional and even self-explanatory. :wink: 

 - Example of a good PR title
    _**Hello Microverse Project Setup**_

   ### HOW TO EDIT PR TITLE
    ![chrome-capture-2023-3-27 (1)](https://user-images.githubusercontent.com/87197412/234717588-992cb1ef-8a21-4987-ac09-5c833bfabc41.gif)

- [ ] Kindly make sure to remove all comments and make sure to customize the content of the README and provide unique information according to your project in all sections and remove all text placeholders which will make your repo clean and professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure to write your commit message in the imperative tense. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Rysth commented:
Thanks! I already improved my README and Title for this first PR. So I hope everything it's okey. Please, let me know if something if not well defined.

Shedrack-Sunday commented on README.md:
```diff
@@ -265,8 +148,6 @@ I would like to thank...
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
- [ ] Kindly add this file `License` to the root of your project so it does not show an error when clicked on
- Follow this [Link](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and customise the content for the license file 👍

Rysth replied:
Done!

Shedrack-Sunday commented on README.md:
```diff
@@ -228,33 +122,22 @@ Feel free to check the [issues page](../../issues/).
-
 - **[Question_1]**
 
   - [Answer_1]
```
- [ ] For the `FAQ` section, it's the only optional portion of your README. kindly add possible user questions and answers, or delete the section in its entirety 👍

Shedrack-Sunday commented on README.md:
````diff
@@ -70,156 +42,78 @@ correctly in every project.
-```
- -->
-
 ### Setup
 
 Clone this repository to your desired folder:
 
````
- [ ] Under this section, kindly give direcions on how to clone your project and use it.
- For example:

```
cd folder
git clone <your repo name>
code . (command to open on vscode)
```

LionRouge1 replied:
- [ ] As mentioned by the previous reviewer, kindly add the steps to run the project locally. That will make your project look more professional.
```sh
git clone <your repo name>
cd  <folder>
```

Shedrack-Sunday reviewed (changes requested):
Hello @Rysth     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯

![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights   
- Good use of GitHub features ✔️  
- Kudus making changes suggested previously by the last reviewer 👍 
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

LionRouge1 reviewed (changes requested):
Hi @Rysth, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost](https://media.giphy.com/media/R2m2NzVxQ3pbG/giphy.gif)

## Suggested changes :recycle: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Rysth commented:
Thanks! I already finished a few things on my last commit: 

- Fixed README.md document.
- Removed OPTIONAL items from the README.md file.
- Fixed the LICENSE link to my MIT.md file.

Alusp commented on README.md:
````diff
@@ -44,182 +16,71 @@ After you're finished please remove all the comments and instructions!
-```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
-
````
- [ ] [Required] Great job implementing the changes :clap:, However, As requested by the previous reviewer Please add the steps on how to run the project locally from your repository not to remove it. because When other developers join your project or when you open-source your project, providing clear instructions on how to run the project locally helps them get up and running quickly. It reduces the time they spend on figuring out dependencies, configurations, and setups, allowing them to start working on the project faster. :+1:  

- Please add the following to your readme file and edit it accordingly. :arrow_down: 
> cd  `<folder>`
git clone `<your repo name>`


- Check out this example below. :arrow_down: 
![image](https://github.com/Rysth/Hello-Microverse-Project/assets/43528594/3edbca44-e3bd-4982-bfb8-524d81bfc846)

Rysth replied:
Thanks! I think that's it's everything fine right now.

Alusp reviewed (changes requested):
Hi @Rysth, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so Close](https://media.giphy.com/media/3ogwFKH8NPzxxIl2Cc/giphy.gif)

 ## Highlights

- [ ] Correct git-flow. :white_check_mark: 
- [ ] No linter errors. :white_check_mark: 
- [ ] Well-detailed PR summary. :white_check_mark:  


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Johnadibe reviewed (approved):
# Status: Approved ✔️
### Hi @Rysth👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![LeonardoInGatsby](https://user-images.githubusercontent.com/38088818/233992044-ad00b20f-3f2f-4c86-8b94-1e613432af5f.gif)

## Highlights
- No linter errors✔️
- Descriptive PR✔️
- Professional README documentation✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
